From 469fb2284363ae2bed42605bb47427c78adbfdd6 Mon Sep 17 00:00:00 2001
From: aaronw <aaron.w2@gmail.com>
Date: Thu, 6 Jun 2019 18:54:17 -0700
Subject: [PATCH 0932/1239] mmc: octeontx: Remove debug message

A debug message was improperly marked as an error message.

Change-Id: Iccdbf468fddfcd96199d2af84e37972e91b63637
Signed-off-by: aaronw <aaron.w2@gmail.com>
Reviewed-on: https://sj1git1.cavium.com/10625
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
---
 drivers/mmc/octeontx_hsmmc.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/mmc/octeontx_hsmmc.c b/drivers/mmc/octeontx_hsmmc.c
index 6ce7d36018..9bb7735798 100644
--- a/drivers/mmc/octeontx_hsmmc.c
+++ b/drivers/mmc/octeontx_hsmmc.c
@@ -1585,11 +1585,11 @@ static int octeontx_mmc_adjust_tuning(struct mmc *mmc, struct adj *adj,
 		for (count = 0; count < 2; count++) {
 			err = adj->test(mmc, opcode, NULL);
 			if (err) {
-				pr_err("%s(%s, %s): tap %d failed, count: %d, rsp_sts: 0x%llx, rsp_lo: 0x%llx\n",
-				       __func__, mmc->dev->name,
-				       adj->name, tap, count,
-				       read_csr(mmc, CAVM_MIO_EMM_RSP_STS()),
-				       read_csr(mmc, CAVM_MIO_EMM_RSP_LO()));
+				pr_debug("%s(%s, %s): tap %d failed, count: %d, rsp_sts: 0x%llx, rsp_lo: 0x%llx\n",
+					 __func__, mmc->dev->name,
+					 adj->name, tap, count,
+					 read_csr(mmc, CAVM_MIO_EMM_RSP_STS()),
+					 read_csr(mmc, CAVM_MIO_EMM_RSP_LO()));
 				break;
 			}
 		}
-- 
2.29.0

